scripts/tests/twister_blackbox/test_verbose.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+"""
+Blackbox tests for twister's command line functions
+"""
+
+import importlib
+import mock
+import os
+import pytest
+import sys
+import re
+
+from conftest import ZEPHYR_BASE, testsuite_filename_mock
+from twisterlib.testplan import TestPlan
+
+
+@mock.patch.object(TestPlan, 'TESTSUITE_FILENAME', testsuite_filename_mock)
+class TestTwisterCommandLine:
+    @classmethod
+    def setup_class(cls):
+        apath = os.path.join(ZEPHYR_BASE, 'scripts', 'twister')
+        cls.loader = importlib.machinery.SourceFileLoader('__main__', apath)
+        cls.spec = importlib.util.spec_from_loader(cls.loader.name, cls.loader)
+        cls.twister_module = importlib.util.module_from_spec(cls.spec)
+
+    @classmethod
+    def teardown_class(cls):
+        pass
+
+    @pytest.mark.usefixtures("clear_log")
+    def test_single_v(self, capfd):
+        args = ['-p', 'qemu_x86', '-T', 'samples/hello_world', '-v']
+
+        with mock.patch.object(sys, 'argv', [sys.argv[0]] + args), \
+            pytest.raises(SystemExit) as sys_exit:
+            self.loader.exec_module(self.twister_module)
+
+        #look for the line INFO - 1/1 qemu samples/hello_world/sample.basic.helloworld PASSED (qemu 2.071s)
+        info_regex = r'INFO\s+-\s+(\d+/\d+)\s+(\S+)\s+(\S+)\s+(\S+)\s+\((\w+)\s+([\d.]+)s\)'
+        info_line_regex = r'^\s*(INFO)'
+
+        out, err = capfd.readouterr()
+        sys.stdout.write(out)
+        sys.stderr.write(err)
+
+        assert re.search(info_regex, err) is not None
+        assert re.search(info_line_regex, err) is not None
+        assert str(sys_exit.value) == '0'
```

Comment:
Move to the front - if the twister command failed as a whole, we want that error to be seen.